app/src/main/java/net/skyscanner/backpack/demo/stories/GradientStory.kt

```diff
@@ -17,7 +17,7 @@ class GradientStory : Story() {
 
   override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
-    gradient_primary.background = BpkGradients.getPrimary(context!!)
-    gradient_direction.background = BpkGradients.getPrimary(context!!, GradientDrawable.Orientation.TR_BL)
+    gradient_primary.background = BpkGradients(context!!)
```

Comment:
Have you considered using `requireContext()` to avoid `!!`? Not sure if it's available in views though.

Also, is it OK to crash on null context?

Reply:
Thanks for the`requireContext()` tip. I will update at multiple places in a different PR.

Reply:
+1 for `requireContext()`